File: src/main/java/bot/commands/ping/PingCommand.java
package bot.commands.ping;

import bot.commands.Command;
import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
import org.jetbrains.annotations.NotNull;

public final class PingCommand extends Command {
    public PingCommand() {
        super(false, "ping", "Pings to Bot to check the Connection!", "");
    }

    @Override
    public void handle(@NotNull MessageReceivedEvent event) {
        event.getChannel().sendMessage("Pong -- Response in *0ms*!").queue();
    }
}

File: src/main/java/bot/commands/ping/PingCommand.kt
package bot.commands.ping

import bot.Bot
import bot.commands.Command
import net.dv8tion.jda.api.events.message.MessageReceivedEvent

class PingCommand(bot: Bot) : Command(bot, false, "ping", "Pings to Bot to check the Connection!", "") {
    override fun handle(event: MessageReceivedEvent) {
        event.channel.sendMessage("Pong -- Response in *0ms*!").queue()
    }
}